makeservices/makemysql-real

```diff
@@ -78,43 +80,50 @@ def main():
         # Read config file.
         mysql_host, mysql_root_pw = read_config()
 
+        # Added a simple and stupid argument parsing so that other scripts can use it without tunneling in yes.
+        if len(sys.argv) > 1 and sys.argv[1] in ['-s', '--silent']:
```

Comment:
Personally, I'd use `argparse`.

Reply:
There's only one possible argument which I don't want to utilize a heavy parser